app/content/views/upload.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from sentry_sdk import capture_exception
+
+from app.common.azure_blob_storage_handler import uploadBlob
+from app.common.permissions import IsMember
+
+
+@api_view(["POST"])
+@permission_classes([IsMember])
+def upload(request):
+    """ Method for accepting company interest forms from the company page """
+    try:
+        if len(request.FILES) > 1:
```

Comment:
Jeg liker godt å legge sånne sjekker inn i egne variabler for å gjøre det mer lesbart. Denne kan godt legges inn i en variabel
`has_multiple_files = len(request.FILES) > 1`